packages/cozy-sharing/src/components/ShareByEmail.jsx

```diff
@@ -126,7 +126,9 @@ export class ShareByEmail extends Component {
       })
 
       Alerter.success(
-        ...getSuccessMessage(recipientsBefore, contacts, documentType)
+        this.props.t(
```

Comment:
This never worked properly but cozy-drive used to display its own translation so the bug never was detected.

Reply:
It works correctly! `Alerter` can accept a `t` props https://github.com/cozy/cozy-ui/blob/master/react/Alerter/index.jsx#L113 

It was done in Drive https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/layout/Layout.jsx#L30

So there is no bug. 

If translation is now handled in cozy-sharing, we'll have an error in the console saying that the key will not exist but will be correctly translated.  

BUT the component should not be linked to translate() anymore but to withLocal

Reply:
there must be some problem elsewhere then, in cozy-notes the message was never translated despite receiving the correct parameters (email, locale key)

Reply:
In Notes, there is no `t` passed to Alerter https://github.com/cozy/cozy-notes/blob/81081c2e5f9c3401e39daba4cf76b68fab615b1b/src/components/app.jsx#L119